src/main/java/com/example/todolist/Domain/Entity/User.java

```diff
@@ -24,6 +26,9 @@ public class User extends BaseEntity {
     @PhoneRule
     private String phone;
 
+    @OneToMany(mappedBy = "user")
```

Comment:
public it, this should be nullable, and make sure to ignore tasks when we fetch the user